### PR TITLE
Update multiline-components.md

### PR DESCRIPTION
**Pull request by AmandaRiu**
Remove fixed items from the Known Issues section.

### CODE REVIEW

File: test-cases/gutenberg/writing-flow/multiline-components.md [hunk 1]
```diff
@@ -12,10 +12,7 @@ Test the next steps on:
 
 **Known Issues**
 - On Android, tapping Enter to split a quote or pullquote citation both splits the block and adds a newline to the citation ([#2498](https://github.com/wordpress-mobile/gutenberg-mobile/issues/2498))
-- On Android, pressing enter multiple times in the content of a Pullquote or Quote block causes unintended behavior ([#27690](https://github.com/WordPress/gutenberg/issues/27690))
 - On Android, there is sometimes a loss of lines when entering multiple lines. ([#29861](https://github.com/WordPress/gutenberg/issues/29861))
-- There is sometimes inconsistent HTML surrounding newlines in the Pullquote and Quote Block.  ([#1396](https://github.com/WordPress/gutenberg/issues/1396))
```
Comment (SiobhyB) [inline test-cases/gutenberg/writing-flow/multiline-components.md:17]: I'm still able to replicate the issues with inconsistent HTML for Pullquote and Quote blocks, but see that the link here is incorrect. It should be linking to the Gutenberg Mobile repository: https://github.com/wordpress-mobile/gutenberg-mobile/issues/1396

@AmandaRiu, are you seeing the issue with the working link?

**Review by SiobhyB — CHANGES_REQUESTED** [review]
I always look up to how diligent you are in your testing and sweeping up things like this @AmandaRiu. :) 

I confirmed that two of the issues are resolved, but am still able to replicate the issue with inconsistent HTML in Quote/Pullquote blocks. The GitHub link isn't correct (it's linking to the Gutenberg repository instead of Gutenberg Mobile) which I think may have led to the proposed deletion here.

I think it makes sense to keep that specific bullet but update the link, let me know what you think!

File: test-cases/gutenberg/writing-flow/multiline-components.md (hunk 1 above)
Comment (AmandaRiu) [reply test-cases/gutenberg/writing-flow/multiline-components.md:17]: Ah...ok, that makes sense. I just opened the link and saw that it was closed and assumed it was fixed. I should've tested it and then would've realized the same thing. I'll get that added back in.

Comment (AmandaRiu) [reply test-cases/gutenberg/writing-flow/multiline-components.md:17]: Fixed in 3e93b932df6e8e16b70f01e61fea0843bd4536d2. Ready for another round. Thanks @SiobhyB !

**Review by SiobhyB — APPROVED** [review]
LGTM :shipit:

**Pull request merged**